cpp/bdayGift.cpp

```diff
@@ -0,0 +1,25 @@
+#include <cmath>
+#include <cstdio>
+#include <vector>
+#include <iostream>
+#include <iomanip>
+#include <algorithm>
+using namespace std;
+
+int main() {
+    int N;
+    cin >> N;
+
+    cout << fixed;
+    cout << setprecision(1);
+    double total = 0;
+    for (int i = 0; i < N; i++) {
+        int cur;
+        cin >> cur;
+        total += 0.5 * cur;
+    }
+
+    cout << total << endl;
+
+    return 0;
+}
```

Comment:
__nitpick__ There should always be an endline `\n` character at the end of very file when you commit something. This is like a good software development todo that the industry follows. so just press enter and add an empty line here. thats it :)

Reply:
Issue Resolved